implementations/rust/ockam/ockam_api/src/logs/setup.rs

```diff
@@ -135,9 +135,6 @@ impl LoggingTracing {
             result.expect("Failed to initialize tracing subscriber");
         };
 
-        // the global error handler prints errors when exporting spans or log records fails
-        set_global_error_handler(logging_configuration);
```

Comment:
Is this indented?

Reply:
Nope, thanks for catching that. It's a leftover from something I was experimenting with